client/layout/masterbar/style.scss

```diff
@@ -392,7 +392,7 @@ $autobar-height: 20px;
 
 	.is-group-editor & {
 		background: var( --masterbar-toggle-drafts-editor-background );
-		border-left: 1px solid var( --masterbar-toggle-drafts-editor-border );
+		border-left: 1px solid var( --masterbar-toggle-drafts-editor-border-color );
```

Comment:
Would there ever be ambiguity between the _thing_ and the _property_? I'm thinking something similar to BEM where a `--` would separate the _thing_ and the _property_.

`--masterbar-toggle-drafts-editor-border-color` might become `--masterbar-toggle-drafts-editor-border--color`

Just thinking out loud I'm totally happy with what we have here already :)